Campaign_Manager/tests.py

```diff
@@ -1,3 +1,81 @@
 from django.test import TestCase
+from .models import *
+from Character_Builder.models import *
 
-# Create your tests here.
+# Campaign test cases
+class CampaignCreateTestCase(TestCase):
+	def setUp(self):
+		Campaign.objects.create(campaignName="The Mountain")
+
+	def test_campaign_name(self):
+		campaign = Campaign.objects.get(campaignName="The Mountain")
+		self.assertEqual(campaign.campaignName, "The Mountain")
+
+	def test_campaign_id(self):
+		campaign = Campaign.objects.get(campaignName="The Mountain")
+		self.assertEqual(campaign.campaignID, 1)
+
+# CampaignDM test cases
+class CampaignDMCreateTestCase(TestCase):
+	def setUp(self):
```

Comment:
use the variables defined in here. You reassign char, dm, and camp in subsequent tests

Reply:
these are locally created function variables and cannot be used by other test methods